File: Hospital.Server/Entities/Response/LabOrderItemResponse.cs
namespace Hospital.Server.Entities.Response
{
    public class LabOrderItemResponse
    {
        public long Id { get; set; }
        public long LabOrderId { get; set; }
        public long LabExamId { get; set; }
        public string ExamName { get; set; } = string.Empty;
        public decimal Amount { get; set; }
        public string? ResultValue { get; set; }
        public string? ResultUnit { get; set; }
        public string? ReferenceRange { get; set; }
        public bool IsOutOfRange { get; set; }
        public string? ResultNotes { get; set; }
        public string? ResultDate { get; set; }
        public bool IsPublished { get; set; }
        public int State { get; set; }
        public string CreatedAt { get; set; } = string.Empty;
        public long CreatedBy { get; set; }
        public long? UpdatedBy { get; set; }
        public string? UpdatedAt { get; set; }
    }
}

File: hospital.client/src/types/LabOrderItemResponse.ts
export interface LabOrderItemResponse {
  id: number;
  labOrderId: number;
  labExamId: number;
  examName?: string | null;
  amount?: number | null;
  resultValue?: string | null;
  resultUnit?: string | null;
  referenceRange?: string | null;
  isOutOfRange?: boolean | null;
  resultNotes?: string | null;
  resultDate?: string | null;
  isPublished?: boolean | null;
  state: number;
  createdAt: string;
  createdBy: number;
  updatedBy?: number | null;
  updatedAt?: string | null;
}

export interface LabOrderItemRequest {
  id?: number | null;
  labOrderId?: number | null;
  labExamId?: number | null;
  examName?: string | null;
  amount?: number | null;
  resultValue?: string | null;
  resultUnit?: string | null;
  referenceRange?: string | null;
  isOutOfRange?: boolean | null;
  resultNotes?: string | null;
  resultDate?: string | null;
  isPublished?: boolean | null;
  state?: number | null;
  createdBy?: number | null;
  updatedBy?: number | null;
}
